apps/barnepensjon-ui/src/hooks/useCountries.ts

```diff
@@ -22,34 +21,38 @@ interface Country {
     }
 }
 
-export const moveMostUsedCountriesToBeginning = (allCountries: Country[]) => {
-    const frequentlyUsed = ['NORGE']
+export const moveNorwayToBeginning = (allCountries: Country[]) => {
```

Comment:
Tanken her var å legge til flere av de mest brukte landene, men det har ikke blitt prioritert å finne ut de neste på listen så det er vell sikkert greit å bare skrive om til å kun flytte Norge opp

Reply:
Funskjonaliteten der var litt wonkey, for den flyttet ikke norge til starten. Den la til en ny norge til starten

Det + at combox dobler opp duplikate options hver gang man åpner og lukker så gjorde jeg det slik for nå